internal/v5/resources.go

```diff
@@ -161,11 +138,63 @@ func (h *ReqHandler) metaResources(entity *mongodoc.Entity, id *router.ResolvedU
 	return results, nil
 }
 
+// GET id/meta/resource
+// https://github.com/juju/charmstore/blob/v5/docs/API.md#get-idmetaresources
+func (h *ReqHandler) metaResourcesSingle(entity *mongodoc.Entity, id *router.ResolvedURL, path string, flags url.Values, req *http.Request) (interface{}, error) {
+	data, err := h.metaResourcesSingle0(entity, id, path, flags, req)
+	if err != nil {
+		if errgo.Cause(err) == params.ErrNotFound {
+			logger.Infof("replacing not-found error on %s/meta/resources%s: %v (%#v)", id.URL.Path(), path, err, err)
```

Comment:
I think Info is appropriate - we log all errors returned from the API and here we're discarding some of that info that would usually be logged, so I think it's useful to have, even in a non-debug context.